mu4/cloud/qml/MuseScore/Cloud/AuthorizationContent.qml

```diff
@@ -0,0 +1,117 @@
+import QtQuick 2.7
+import QtQuick.Layouts 1.15
+import MuseScore.UiComponents 1.0
+import MuseScore.Cloud 1.0
+
+FocusScope {
+    id: root
+
+    signal createAccountRequested()
+    signal signInRequested()
+
+    Rectangle {
+        anchors.fill: parent
+        color: ui.theme.backgroundColor
+    }
+
+    QtObject {
+        id: privateProperties
+
+        readonly property int sideMargin: 134
+        readonly property int buttonWidth: 200
+    }
+
+    StyledTextLabel {
+        id: pageTitle
+
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.topMargin: 66
+        anchors.leftMargin: privateProperties.sideMargin
+
+        text: qsTrc("scores", "Account")
+
+        font.pixelSize: 32
+        font.bold: true
+    }
+
+    Image {
+        id: logo
+
+        anchors.top: pageTitle.bottom
+        anchors.topMargin: 44
+
+        height: 240
+        width: parent.width
+
+        visible: root.height > 600
+
+        source: "resources/mu_logo_background"
```

Comment:
Where are these resources located?

Reply:
found

Reply:
This 1.2 MB file is really that big? It can take a long time to load

Reply:
I think if this is a problem, we can change the background in the future. I took these assets from figma

Reply:
How can we find out if this is a problem or not?
Maybe you checked how quickly the page opens on weak computers?
I really don't understand why to use a 2560x1600 picture, load it and resize it to a small size. I think you need to ask the designer.

Reply:
I asked the designer about this background. She answered that is a temporary solution and background will be changed in the future. I scaled the current background, so it has not such large size

Reply:
That's a relief. Thanks, @RomanPudashkin.

Reply:
I do wonder if it will work on 5.9, perhaps you should specify the format of file (.svg, .jpeg, etc.)

Reply:
fixed